smatch/matcher.py

```diff
@@ -64,15 +64,15 @@ def sphdist(lon1, lat1, lon2, lat2):
     vec1 = _lonlat2vec(lon1, lat1)
     vec2 = _lonlat2vec(lon2, lat2)
 
-    cosd = (
-        vec1[..., 0] * vec2[..., 0]
-        + vec1[..., 1] * vec2[..., 1]
-        + vec1[..., 2] * vec2[..., 2]
+    dsq = (
```

Comment:
You might link this SO post: https://stackoverflow.com/questions/52210911/great-circle-distance-between-two-p-x-y-z-points-on-a-unit-sphere